### PR TITLE
docs(CHANGELOG): stage unreleased changes 

### PR DESCRIPTION
**Pull request by tassja**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ All notable changes to PlaceOS are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PlaceOS Platform Versioning](./README.md#platform-versioning).
 
+## Unreleased
+
+### Added
+- Get access token using resource owner's credentials (insecure) in `crystal-client`
+- Granular OAuth Scopes in `rest-api`
+- Support for nest hashes in `source` influx publisher
```
Comment (caspiano) [inline CHANGELOG.md]:
```suggestion
- Support for nested hashes in `source` influx publisher
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,21 @@ All notable changes to PlaceOS are documented in this file.
...
+
+### Fixed
+- Handle proxied PUBLISH events in `edge`
+- Resolved issues with multi-type fields in `rubber-soul` table manager
+- Add validation for booking times in `staff-api`
+- Look up events with system mailbox in `staff-api`
+- Able to mark visitor as unexpected in `staff-api`
+- Allow tenant conference_type to be configurable in `staff-api`
+
```
Comment (caspiano) [inline CHANGELOG.md:20]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,21 @@ All notable changes to PlaceOS are documented in this file.
...
+- Get access token using resource owner's credentials (insecure) in `crystal-client`
+- Granular OAuth Scopes in `rest-api`
+- Support for nest hashes in `source` influx publisher
+
+### Fixed
+- Handle proxied PUBLISH events in `edge`
+- Resolved issues with multi-type fields in `rubber-soul` table manager
+- Add validation for booking times in `staff-api`
+- Look up events with system mailbox in `staff-api`
+- Able to mark visitor as unexpected in `staff-api`
```
Comment (caspiano) [inline CHANGELOG.md]:
```suggestion
- Ability to mark visitor as unexpected in `staff-api`
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,21 @@ All notable changes to PlaceOS are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PlaceOS Platform Versioning](./README.md#platform-versioning).
 
+## Unreleased
+
+### Added
+- Get access token using resource owner's credentials (insecure) in `crystal-client`
```
Comment (caspiano) [inline CHANGELOG.md]: Perhaps this is a bit too internal for public changelog?

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,21 @@ All notable changes to PlaceOS are documented in this file.
...
+- Granular OAuth Scopes in `rest-api`
+- Support for nest hashes in `source` influx publisher
+
+### Fixed
+- Handle proxied PUBLISH events in `edge`
+- Resolved issues with multi-type fields in `rubber-soul` table manager
```
Comment (caspiano) [inline CHANGELOG.md]: Too internal, perhaps we should do

```suggestion
- Resolved issues with Elasticsearch indexing
```

**Review by w-le — APPROVED** [review]
lgtm

**Review by caspiano — APPROVED** [review]

**Pull request merged**